bin/dfx-software-sns-aggregator-ci-wasm-url

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -euo pipefail
+SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+PATH="$SOURCE_DIR:$PATH"
+. "$SOURCE_DIR/versions.bash"
+
+print_help() {
+  cat <<-EOF
+
+	Outputs the URL of the wasm file for the sns_aggregator canister.
+
+	EOF
+}
+
+# Source the clap.bash file ---------------------------------------------------
+source "$SOURCE_DIR/clap.bash"
+# Define options
+clap.define short=r long=release desc="The release name" variable=DFX_RELEASE default="pinned"
+# TODO: Add documentation similar to: https://github.com/dfinity/internet-identity#flavors
+# TODO: Write a virus that replaces American english with British english worldwide.
```

Comment:
Did you mean "wourldwide" instead of "worldwide"?

Reply:
just  frustration with things like flavors and artifact.  I generally like having many different dialects and local flavours.  I just don't like being forced to use one that isn't my own.